### PR TITLE
fixing background display bug

### PR DESCRIPTION
**Pull request by lecunm**

### CODE REVIEW

File: app/services/PageService.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ case class DirectoryMeta(label: Option[String] = None, description: Option[Strin
 
 case class TagsModule(tags: Seq[String])
 
-case class ScenariosModule(project: Option[String] = None, branchName: Option[String] = None, feature: Option[String] = None, select: Option[TagsModule] = None)
+case class ScenariosModule(project: Option[String] = None, branchName: Option[String] = None, feature: Option[String] = None, select: Option[TagsModule] = None, includeBackground: Option[Boolean] = None)
```
Comment (notflorian) [inline app/services/PageService.scala]: Where do you set includeBackground?

File: app/services/PageService.scala [hunk 2]
```diff
@@ -303,7 +303,8 @@ class PageService @Inject()(config: Configuration, projectRepository: ProjectRep
             fragmentUnderProcessing.scenariosModule match {
               case Some(scenariosModule) =>
                 val feature = buildFeature(scenariosModule, pageJoinProject)
-                PageFragmentUnderProcessing(scenarios = feature)
+                val background = if(scenariosModule.includeBackground.getOrElse(false)) feature.map(_.background) else None
+                PageFragmentUnderProcessing(scenarios = feature.map(_.copy(background = background.getOrElse(None))))
```
Comment (notflorian) [inline app/services/PageService.scala]: useless getOrElse(None)

File: app/services/PageService.scala [hunk 3]
```diff
@@ -303,7 +303,8 @@ class PageService @Inject()(config: Configuration, projectRepository: ProjectRep
             fragmentUnderProcessing.scenariosModule match {
               case Some(scenariosModule) =>
                 val feature = buildFeature(scenariosModule, pageJoinProject)
-                PageFragmentUnderProcessing(scenarios = feature)
+                val background = if(scenariosModule.includeBackground.getOrElse(false)) feature.map(_.background) else None
```
Comment (notflorian) [inline app/services/PageService.scala]:
```suggestion
                val background = scenariosModule.includeBackground.flatMap(feature.map(_.background))
```

File: app/services/PageService.scala (hunk 1 above)
Comment (lecunm) [reply app/services/PageService.scala]: In the scenario inclusion see https://github.com/KelkooGroup/theGardener/issues/64

**Pull request merged**